sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryStorageWriteApiSchemaTransformProvider.java

```diff
@@ -157,24 +160,41 @@ public static Builder builder() {
           .Builder();
     }
 
+    @SchemaFieldDescription(
+        "The bigquery table to write to. Format: [${PROJECT}:]${DATASET}.${TABLE}")
     public abstract String getTable();
 
+    @SchemaFieldDescription(
+        "Optional field that specifies whether the job is allowed to create new tables. "
+            + "The following values are supported: CREATE_IF_NEEDED (the job may create the table), CREATE_NEVER ("
+            + "the job must fail if the table does not exist already).")
     @Nullable
     public abstract String getCreateDisposition();
 
+    @SchemaFieldDescription(
+        "Specifies the action that occurs if the destination table already exists. "
+            + "The following values are supported: "
+            + "WRITE_TRUNCATE (overwrites the table data), "
+            + "WRITE_APPEND (append the data to the table), "
+            + "WRITE_EMPTY (job must fail if the table is not empty).")
```

Comment:
Could we also add the default here too if the value isn't specified? I believe it should be WRITE_APPEND based on this PR: https://github.com/apache/beam/pull/25434